jsapp/xlform/src/model.row.coffee

```diff
@@ -235,15 +250,17 @@ module.exports = do ->
       begin_xlsformrow['kobo--rank-items'] = @getList().get('name')
       survey_arr.push(begin_xlsformrow)
       ``
+      return
 
   class ScoreChoiceList extends Array
 
   class ScoreRow extends SimpleRow
     initialize: ->
       @set('type', 'score__row')
       @simpleEnsureKuid()
+      return
     export_relevant_values: (surv, sheets)->
-      surv.push(@attributes)
+      return surv.push(@attributes)
```

Comment:
In all the other `export_relevant_values` we don't return anything. And AFAIK we only call this function in one place and it is not being assigned to anything